script/libtest.bash

```diff
@@ -149,7 +149,7 @@ _wait_for_kubeless_kafka_server_ready() {
     echo_info "Waiting for kafka-0 to be ready ..."
     k8s_wait_for_pod_logline "Kafka.*Server.*started" -n kubeless kafka-0
     sleep 10
-    kubeless topic create "${test_topic}"
+    kubeless topic create "${test_topic}" || true
```

Comment:
@jjo @ngtuna for now I'm allowing this line to fail, this will allow us to ignore the topic already exists error and if the topic fails to create the first time the next line will fail waiting for it to appear.

Other options:
- do we actually need this test topic? We already wait in https://github.com/kubeless/kubeless/blob/master/script/libtest.bash#L240 for the function's topic to be created
- add a random string to the end of `test-sentinel` so we create and wait multiple times for this
- delete the topic after the test (not sure if this is reliable because it actually will get 'marked for deletion' and not immediately deleted)

Reply:
Thanks @prydonius for investigating it. I vote to go with the first option :-)

Reply:
@ngtuna did you mean the `|| true` or removing the creation of the sentinel topic? @jjo would like to get your take

Reply:
Sorry I will make my point clear. Basically I see we are checking the kafka-ready state in every pubsub tests, and I assume we are all agree on this point. So the main question is: should we create a test topic at every tests or should we rely on the particular topic creation from the pubsub func itself ? So I vote to go with the first "bullet" from @prydonius 's suggestion that we can just remove the `test-centinel` topic as I see there is a duplicated action once the pubsub func create its topic

Reply:
IMO we should instead record the already verified state, and immediately succeed,
this will also avoid going thru the logtailing and whatnot again.
I've implemented this at https://github.com/prydonius/kubeless/pull/1
against adnan's branch.

Reply:
👍 Can you check this @prydonius ?

Reply:
> IMO we should instead record the already verified state, [...]
Not possible via env var, as these are run from totally diff processes,
would need a state file or alike.
+1 the `|| true` sp to avoid blocking release, I'll see if I can
improve it in a follow up PR. <- @prydonius @ngtuna